modules/rostests/apitests/crt/crtdata.c

```diff
@@ -532,9 +553,14 @@ void Test__wenviron(void)
 #ifdef _M_IX86
     ok_ptr(__p__wenviron(), &_wenviron);
 #endif
-#if (WINVER >= 0x600)
-    _CRTIMP unsigned int __cdecl _get_wenviron(void);
-    ok_int(_get_wenviron(), _wenviron);
+
+#if 0 // FIXME: This returns an error / NULL on Windows 10
```

Comment:
does the ansi version (`environ`) do that too?

Reply:
No, see line 346, which passes. Maybe the executable isn't "unicoded"? Maybe the startup code has a bug? Suggestions welcome.